dkam/internal/env/env.go

```diff
@@ -10,15 +10,17 @@ import (
 )
 
 const (
-	envHookOSRepo    = "RS_HOOK_OS_REPO"
-	envHookOSVersion = "HOOK_OS_VERSION"
+	envUOSRepo                 = "UOS_REPO"
+	envUOSVersion              = "UOS_VERSION"
+	envNameRsFilesProxyAddress = "RSPROXY_FILES_ADDRESS"
 )
 
 var (
 	// TODO: pass all hook os version and bare metal agent
 	//  versions via configmap or override values to dkam.
-	HookOSRepo    = os.Getenv(envHookOSRepo)
-	HookOSVersion = os.Getenv(envHookOSVersion)
+	UOSRepo        = os.Getenv(envUOSRepo)
+	UOSVersion     = os.Getenv(envUOSVersion)
+	RSProxyAddress = os.Getenv(envNameRsFilesProxyAddress)
```

Comment:
can we not get this from the infra-config map?

Reply:
Yes we should get this from infra-config.